### DISCUSSION TITLE
Barsuk wreck lacks links for guns preventing thalamus shooting

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#14129] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/FakeFishGames/Barotrauma/assets/42521948/ef4adec5-f2a9-4729-abdc-14d1eee28023)

### Reproduction steps

1. open sub editor
2. see da wreck with links turned on

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I talked with one of our designers and this is actually intentional: this is supposed to be a "beginner wreck" you encounter early in the game, and not intended to shoot at the sub.
